modern/src/login/LoginPage.js

```diff
@@ -60,6 +60,8 @@ const LoginPage = () => {
   const registrationEnabled = useSelector((state) => state.session.server.registration);
   const languageEnabled = useSelector((state) => !state.session.server.attributes['ui.disableLoginLanguage']);
   const emailEnabled = useSelector((state) => state.session.server.emailEnabled);
+  const oidcEnabled = useSelector((state) => state.session.server.oidcEnabled);
+  const oidcForced = useSelector((state) => state.session.server.oidcEnabled && state.session.server.oidcForce);
```

Comment:
Also need to rename these to `openId` instead of `oidc`. Related to backend comment.